app/assets/javascripts/preferences/components/Pane.tsx

```diff
@@ -16,12 +16,14 @@ export const PreferencesGroup: FunctionComponent = ({ children }) => (
   <div className="bg-default border-1 border-solid rounded border-gray-300 px-6 py-6 flex flex-col gap-2">
     {!Array.isArray(children)
       ? children
-      : children.map((c, i, arr) => (
-          <>
-            {c}
-            <HorizontalLine index={i} length={arr.length} />
-          </>
-        ))}
+      : children
+          .filter((c) => !!c)
```

Comment:
I've changed the filter to `c!=null && c!==''` to be more descriptive. Basically since we're putting a HorizontalLine between segments, and the children array contains null children it'll put a horizontal line between them because they're members of the array.